rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/330_fetch_fields.yml

```diff
@@ -124,10 +124,9 @@ setup:
       search:
         index: test
         body:
-          fields: [keyword]
+          fields: [text]
   - match: { error.root_cause.0.type: "illegal_argument_exception" }
-  - match: { error.root_cause.0.reason: "Unable to retrieve the requested [fields] since _source is disabled
-        in the mappings for index [test]" }
+  - match: { error.root_cause.0.reason: "error fetching [text]: Unable to retrieve values because _source is disabled in the mappings for index [test]" }
```

Comment:
should we have more tests with _source disabled, I mean some that cover the cases that now work that would have returned error before?